;

```diff
@@ -0,0 +1,16 @@
+Revert "removing former team members from the team page"
```

Comment:
Hello! This file needs to be deleted, as comments regarding reverted commits should not be included in this PR.